20-packages.ks

```diff
@@ -1,40 +1,58 @@
 %packages --excludedocs
-bash
+# from lorax examples
+@core
 kernel
-biosdevname
+kernel-modules
+kernel-modules-extra
+grub2-efi
 grub2
-grub2-tools
-e2fsprogs
-passwd
-policycoreutils
-chkconfig
-rootfiles
-yum
-vim-minimal
-acpid
-tftp
-lldpad
+shim
+syslinux
+dracut-config-generic
+# not needed for ISO
+#dracut-network
+-dracut-config-rescue
+tar
 isomd5sum
-elfutils-libs
-curl
+
+# required by lorax
+dracut-live
+# required by bootloader
+centos-logos
+#syslinux-nonlinux
+memtest86+
+#bash
+#kernel
+biosdevname
+#grub2
+#grub2-tools
+#e2fsprogs
+#passwd
+#policycoreutils
+#chkconfig
+#rootfiles
+#yum
+vim-minimal
+#elfutils-libs
 
 # Dracut missing deps (https://bugzilla.redhat.com/show_bug.cgi?id=1285810)
-tar
-gzip
+#tar
+#gzip
 
 # Facter
-tfm-rubygem-facter
+rubygem-facter
 ethtool
+lldpad
 net-tools
 dmidecode
-bind-utils
 virt-what
 
 # Foreman proxy
 foreman-discovery-image-service
+curl
+wget
```

Comment:
Are these really needed? `smart-proxy` already depends on `curl` and AFAIK `wget` isn't needed. If it is, shouldn't `foreman-discovery-image-service` depend on it?

Reply:
Both are required at the moment, my plan is to get rid of smart proxy from FDI soon, so that plugin will be deprecated. I want to write a small service in go that would provide the three API calls instead so Ruby would only be required for Facter. Actually, this is my Christmas project this year :)